### PR TITLE
Describe more config for running in production

### LINKED ISSUE #109: Update the "Config file for production"-section in README
The "Config file for production"-section must be checked against the actual prod config file in order to find out whether there are gaps in this section that should be filled and/or some variables that should be described.

I have so far found the following gap in the "Config file for production"-section:
- when setting ``[howitz] -> devmode`` to `false`, the app does not run since Flask's `listen` (address) and `port` values are not provided. The (separate) "Run in production"-section refers to flask's docs in order to help configuring the flask app. **But**: one - some info in the "Run in production"- and the "Config file for production"-section belongs together yet it is spread across the `README`; two - referring to Flask docs does not necessarily help with correctly placing the flask related config values in the Howitz' config file.
- ... more?

### PR DESCRIPTION
**Pull request by hmpf**
Closes #109

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -312,7 +312,16 @@ file.
 ``[flask] -> DEBUG`` should be ``false``. You can still override it via an
 environment variable.
 
-``[howitz] -> devmode`` should be ``false``.
+``[howitz] -> devmode`` should be ``false``. When ``devmode`` is ``false`` it
+is necessary to explicitly write in the ``[howitz]`` section what ip-address
```
Comment (johannaengland) [inline README.rst:316]:
```suggestion
is necessary to explicitly write in the ``[howitz]`` section which IP-address
```

File: README.rst [hunk 2]
```diff
@@ -312,7 +312,16 @@ file.
...
+and port to listen on::
+
+    [howitz]
+    devmode = false
+    listen = 127.0.0.1
+    port = 5000
+
+(127.0.0.1 and 5000 are the flask defaults.)
```
Comment (johannaengland) [inline README.rst:324]:
```suggestion
(127.0.0.1 and 5000 are the Flask defaults.)
```

**Comment by hmpf** [comment]
I would much rather have a section in the actual docs for production, with a complete example file. I realized when I was checking our prod setup recently that we HAVE a docs tree. *facepalm*

I added "listen" and "port" to my dev config file in ``[howitz]`` and set devmode to false, and it works. (After a few false starts and wrong locations).

**Comment by hmpf** [comment]
> > I would much rather have a section in the actual docs for production, with a complete example file. I realized when I was checking our prod setup recently that we HAVE a docs tree. _facepalm_
> 
> Sounds great! This PR or separate issue?

Separate, we could have a META "move docs from README to docs/" issue.

**Pull request merged**